src/lang/lt-LT.json

```diff
@@ -178,60 +178,60 @@
   "message.confirm-delete": "Ar esate tikri, jog norite ištrinti svetainę {target}?",
-  "message.confirm-leave": "Are you sure you want to leave {target}?",
-  "message.confirm-remove": "Are you sure you want to remove {target}?",
+  "message.confirm-leave": "Ar esate tikri, jog norite palikti {target}?",
+  "message.confirm-remove": "Ar esate tikri, jog norite ištrinti {target}?",
   "message.confirm-reset": "Are esate tikri, jog norite atstatyti svetainės {target} statistikos duomenis?",
-  "message.delete-team-warning": "Deleting a team will also delete all team websites.",
+  "message.delete-team-warning": "Ištrinant komandą bus ištrintos ir visos komandos svetainės.",
   "message.delete-website-warning": "Visi susiję duomenys taip pat bus ištrinti.",
   "message.error": "Kažkas įvyko ne taip.",
   "message.event-log": "{event} on {url}",
   "message.go-to-settings": "Eiti į nustatymus",
   "message.incorrect-username-password": "Neteisingas vartotojo vardas/slaptažodis.",
   "message.invalid-domain": "Klaidingas domenas",
   "message.min-password-length": "Minimum length of {n} characters",
-  "message.new-version-available": "A new version of Umami {version} is available!",
+  "message.new-version-available": "Išleista nauja 'Umami' {version} versija!",
   "message.no-data-available": "Nėra jokių duomenų.",
   "message.no-event-data": "No event data is available.",
   "message.no-match-password": "Slaptažodžiai nesutampa",
-  "message.no-results-found": "No results were found.",
-  "message.no-team-websites": "This team does not have any websites.",
-  "message.no-teams": "You have not created any teams.",
-  "message.no-users": "There are no users.",
+  "message.no-results-found": "Jokių rezultatų nerasta.",
+  "message.no-team-websites": "Ši komanda neturi jokių svetainių.",
+  "message.no-teams": "Jūs nesate sukūrę jokių komandų.",
+  "message.no-users": "Nėra jokių vartotojų.",
   "message.no-websites-configured": "Jūs nesate susikonfiguravę jokių svetainių.",
   "message.page-not-found": "Puslapis nerastas.",
-  "message.reset-website": "To reset this website, type {confirmation} in the box below to confirm.",
+  "message.reset-website": "Kad atstatyti šią svetainę, įrašykite {confirmation} žemiau, kad patvirtintumėte.",
   "message.reset-website-warning": "Visi šios svetainės statistikos duomenys bus ištrinti, bet sekimo kodas išliks nepaliestas.",
   "message.saved": "Sėkmingai išsaugota.",
   "message.share-url": "Tai yra viešai prieinama {target} nuoroda (URL).",
-  "message.team-already-member": "You are already a member of the team.",
-  "message.team-not-found": "Team not found.",
-  "message.team-websites-info": "Websites can be viewed by anyone on the team.",
+  "message.team-already-member": "Jūs jau esate šios komandos narys.",
+  "message.team-not-found": "Komanda nerasta.",
+  "message.team-websites-info": "Bet kas iš šios komandos gali peržiūrėti svetaines.",
```

Comment:
"websites", as in "websites owned by this team", or "all websites"?

Reply:
That message means team websites.